narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -374,4 +377,13 @@ function process_zip_list_file {
 	done
 }
 
+function remove_txt_file {
+    local zip_list_file="${1}"
+
+	if [[ ! -s "${zip_list_file}" && ! -z "${zip_list_file}" ]]
+	then 
+		rm "${zip_list_file}"
```

Comment:
In scripts, always use 'rm -f', because if one uses an alias, like 'rm="rm -i"', the script will get stuck.